src/transformutil.ts

```diff
@@ -172,20 +173,151 @@ async function dataToJson(file_data: string, opt: any = null) {
 
 async function parseCsv(csvStr: string, options: any) {
     return new Promise<any[]>((res, rej) => {
-        const rows:any[] = [];
+        const rows: any[] = [];
         csv.parseString(csvStr, options)
             .on('data', (row: any) => rows.push(row))
             .on('error', (err: any) => rej(err))
             .on('end', () => res(rows));
     });
 }
 
+export async function getGpxData(node: any, result: any) {
```

Comment:
this function is used locally, can we remove export. Also can we rename this function to getGpxDataFromXmlNode or something like that

Reply:
fixed by roberto-butti in latest push